### ISSUE TITLE
Review Jetty 10 documentation

### ISSUE THREAD

**Issue by sbordet** [eclipse/jetty.project#3164]
As the Jetty 9.4.x documentation is merged into 10.0.x, it retains references to version numbers that are not valid for 10.0.x.

Typical examples are when we report in the documentation output of `java -jar start.jar --list-config`, or where we reference Maven POM snippets, etc.

For example, `jetty-home-and-jetty-base.adoc` reports `servlet-api-3.1.jar` but of course for Jetty 10 that should be changed to `jetty-servlet-api-4.0.0.jar`.

It should be easy to find all places by looking in the documentation for sections that have screen output, and references to known jar names that need an updated version.

**Comment by jmcc0nn3ll** [comment]
IMO, once Jetty 10 lands the documentation on 10 should be canonical and 9 shouldn't be published anywhere online.  It can still be available in the distribution or in the repository as part of the build but in terms of generally accessible through search, I think we do ourselves and the community a disservice having 99.9% of the documentation duplicated and available under separate URLs.  This is one of the things I would like to get cleaned up with the general website cleanup.

**Comment by sbordet** [comment]
@jmcc0nn3ll sure, but what you say is orthogonal with this issue. Jetty 10 documentation needs to be fixed, sooner or later.

**Comment by jmcc0nn3ll** [comment]
fair, though I think things like this will be easier to manage with that in mind

more on topic, it would make sense to centralize these into attrribute strings which are set in the pom.xml and asciidoctor can replace during rendering, there are many examples of this in there

**Comment by gregw** [comment]
@sbordet can we split this issue into 2:
 + restructure documentation - which has to be done before 10.0.0 release
 + fill out documentation - which can be done over time after 10.0.0. release

**Comment by WalkerWatch** [comment]
@gregw +1

We have already split, but work will be ongoing for additions.

**Comment by sbordet** [comment]
It's always in my mind, so we can close this.
